pkg/cmd/cli/describe/describer.go

```diff
@@ -1369,7 +1369,11 @@ func describeBuildTriggerCauses(causes []buildapi.BuildTriggerCause, out *tabwri
 func squashGitInfo(sourceRevision *buildapi.SourceRevision, out *tabwriter.Writer) {
 	if sourceRevision != nil && sourceRevision.Git != nil {
 		rev := sourceRevision.Git
-		formatString(out, "Commit", fmt.Sprintf("%s (%s)", rev.Commit[:7], rev.Message))
+		commit := ""
+		if len(rev.Commit) > 6 {
```

Comment:
is there case when the commit would be >0 but less than 7? If not, we should just check if the commit is not empty and in that case don't print it at all (or just say "empty" or "unknown").